packages/teleport/src/Console/DocumentNodes/DocumentNodes.story.test.tsx

```diff
@@ -28,3 +32,14 @@ test('render DocumentNodes', async () => {
   await waitFor(() => expect(ctx.fetchNodes).toHaveBeenCalledTimes(1));
   expect(container.firstChild).toMatchSnapshot();
 });
+
+test('render DocumentNodes pagination unsupported', async () => {
+  const ctx = createContext(true /* paginationUnsupported */);
```

Comment:
In this situation I'd consider passing an object with options instead of having to document what the boolean means.